include/mbedtls/ssl.h

```diff
@@ -2424,6 +2449,28 @@ typedef int mbedtls_ssl_export_keys_ext_t( void *p_expkey,
                                            const unsigned char client_random[32],
                                            const unsigned char server_random[32],
                                            mbedtls_tls_prf_types tls_prf_type );
+#endif /* MBEDTLS_SSL_PROTO_TLS1 || MBEDTLS_SSL_PROTO_TLS1_1 || \
+          MBEDTLS_SSL_PROTO_TLS1_2 */
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+/**
+ * \brief           Callback type: Export TLS 1.3 secrets (client early traffic
+ *                  secret, client handshake traffic secret, server handshake
+ *                  traffic secret, client application 0 traffic secret, server
+ *                  application 0 traffic secret, exporter master secret)
+ *
+ * \param p_expkey  Context for the callback
+ * \param secret    Pointer to secret
+ * \param len       Secret length
+ *
+ * \return          0 if successful, or
+ *                  a specific MBEDTLS_ERR_XXX code.
```

Comment:
Very minor: We've become more picky with the style of the Doxygen in upstream Mbed TLS, using e.g. full sentences for parameter descriptions, and multiple `\return` statements per documented return value, `\c` around constants etc. 
Feel free to fix if you're bored, or leave it otherwise.